src/Console/ModelsCommand.php

```diff
@@ -332,7 +332,8 @@ protected function castPropertiesType($model)
                     $realType = '\Illuminate\Support\Collection';
                     break;
                 default:
-                    $realType = class_exists($type) ? ('\\' . $type) : 'mixed';
+                    $explodedType = head(explode(':', $type));
+                    $realType = class_exists($explodedType) ? ('\\' . $explodedType) : 'mixed';
```

Comment:
Thanks, I suggest to simplify this; no need for explode and a helper function, [strtok](https://www.php.net/strtok) handles this exactly the way we need; this also means we can keep the original `$realType = ` line:
```suggestion
                    // In case of an optional custom cast parameter , only evaluate
                    // the `$type` until the `:`
                    $type = strtok($type, ':');
                    $realType = class_exists($type) ? ('\\' . $type) : 'mixed';
```

Reply:
thanks for that suggestion